tests/components/elkm1/test_config_flow.py

```diff
@@ -36,6 +46,21 @@
 MODULE = "homeassistant.components.elkm1"
 
 
```

Comment:
We need a test for when the unique ID changes so we don't end up connecting to the wrong system